IowaPD_dispatchtype.yaml

```diff
@@ -12,41 +14,26 @@ entityDefinitions:
       ol.priority:
         type: "ol.priority"
         column: "Type_Priority"
-      datetime.alerted:
-        type: 'datetime.alerted'
-        transforms:
-        - !<transforms.CombineDateTimeTransform>
-          dateColumn: CFS_DateTimeJanet
-          datePattern: ['yyyy-MM-dd HH:mm:ss','yyyy-MM-dd HH:mm:ss.S']
-          timeColumn: TimeDisp
-          timePattern: ['yyyy-MM-dd HH:mm:ss.S']
-          timezone: "America/Chicago"
       datetime.enroute:
```

Comment:
SO for all of these datetimes, we MUST use the CFS datetimeJanet field for the date. The other one is erroneous, but gives the right time. Also we should still include datetime alerted.